tools/selinux/icinga2.sh

```diff
@@ -67,6 +67,7 @@ sepolicy manpage -p . -d icinga2_t
 
 # Label the port 5665
 /sbin/semanage port -a -t icinga2_port_t -p tcp 5665
+/sbin/semanage port -a -t redis_port_t -p tcp 6380
```

Comment:
So this script is for testing purposes only anyways, isn't it? Do we want to keep this in here even though that port labeling will be managed by the new `icinga-selinux-common` package?

Reply:
We just discussed this and will keep it in here.

As this script is for testing without the icinga2-selinux package, it's probably a good idea to label that port as we assume that it is in the policy. Also doesn't really hurt running the port twice, so things won't even break if you do this and install the `icinga-selinux-common` package.